agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -636,6 +636,57 @@ func testV3TaskEndpoint(t *testing.T, taskName, containerName, networkMode, awsl
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
 
+// testContainerMetadataFile validates that the metadata file from the
+// ECS_CONTAINER_METADATA_FILE environment variable contains all the required
+// fields
+func testContainerMetadataFile(t *testing.T, taskName, awslogsPrefix string) {
+	agentOptions := &AgentOptions{
+		ExtraEnvironment: map[string]string{
+			"ECS_ENABLE_CONTAINER_METADATA": "true",
+			"ECS_AVAILABLE_LOGGING_DRIVERS": `["awslogs"]`,
+		},
+	}
+
+	agent := RunAgent(t, agentOptions)
```

Comment:
I think we would still require the right agent version here, since functional tests can be run for different agent images.

Reply:
Gotcha. Fixed it to remain consistent with other tests.